src/mongo/db/encryption/encryption_options.h

```diff
@@ -48,6 +48,12 @@ struct EncryptionGlobalParams {
     std::string vaultServerCAFile;
     bool vaultDisableTLS{false};
     long vaultTimeout{15L};
+    std::string kmipServerName;
+    int kmipPort;
+    std::string kmipServerCAFile;
+    std::string kmipClientCertificateFile;
+    std::string kmipClientKeyFile;
+    std::string kmipKeyIdentifier; // TODO!
```

Comment:
Because I wasn't sure if we wanted to add a default value or not. The enterprise documentation claims that this parameter is optional, but then we would need a reliable way to randomly generate a unique server identifier. The similar variable for vault is also required.